src/js/components/DataTable/StyledDataTable.js

```diff
@@ -107,8 +110,8 @@ const StyledDataTableBody = styled(TableBody)`
     ${focusStyle({ skipSvgChildren: true, forceOutline: true })}
   }
 
-  &:focus:not(:focus-visible) {
-    ${unfocusStyle({ skipSvgChildren: true, forceOutline: true })}
+  &:not(:focus-visible) {
+    ${unfocusStyle()}
```

Comment:
Behavior with keyboard and mouse looks good, but just curious why we needed to adjust this at all. What happens if we leave it as it was before?